### ISSUE TITLE
Kiva build failing on Mac M2

### ISSUE THREAD

**Issue by capn-freako** [enthought/enable#1067]
I'm getting a `Kiva` build failure on my MacBook Air M2 (MacOS 12.7.1):

```
File: /private/var/folders/pf/9gjf96y54f55tdz0vt0wq4kh0000gn/T/pip-install-78pd22fa/enable_d6b6869f6bbd44439ebdfc9c30a63ab0/kiva/_marker_renderer.pxd
        tree = Parsing.p_module(s, pxd, full_module_name)

      Error compiling Cython file:
      ------------------------------------------------------------
      ...
      # Thanks for using Enthought open source!
      import cython
      import numpy as np
      from numpy cimport uint8_t

      cimport _marker_renderer
              ^
      ------------------------------------------------------------

      kiva/_marker_renderer.pyx:14:8: '_marker_renderer.pxd' not found
```

Under `conda-build` I'm able to get past this, by back-reving `Pip` to v22.
But, under `pip install` this trick doesn't work.

**Any suggestions on how to make forward progress, or do further debugging?**

The build completes successfully on the other 3 major platforms:

- Windows 10 64-bit,
- Linux 64-bit, and
- MacOS Intel.

**Comment by corranwebster** [comment]
I _think_ the answer to this is don't use Cython 3: #1060 

There should be Apple Silicon compatible wheels on PyPI, so you ought to be able to install without needing to build for the most recent releases (ie. `pip install` should just work without any build step)

**Comment by jwiggins** [comment]
That's correct. The 6.0.0 release on PyPI does not currently supply Apple Silicon-compatible wheels, only Intel-compatible wheels.

**Comment by itziakos** [comment]
#1070 should fix the build issues on apple silicon (see https://github.com/enthought/enable/actions/runs/15595413632/job/43924580111)

**Comment by capn-freako** [comment]
Whoot! Thank you, @itziakos !
Do you have a tentative date for the next build and release to PyPi?

**Comment by itziakos** [comment]
enable 6.1.0 is already on PyPi (including wheels for apple silicon). Let me know if you have any issues.

**Comment by capn-freako** [comment]
Ah, yes, I see it now. Thanks! :)

Is there a matching v6.1.0 release of Chaco coming soon?

Thanks,
-db

**Comment by itziakos** [comment]
> 
> Is there a matching v6.1.0 release of Chaco coming soon?

Yes, I am working towards a new release

**Comment by capn-freako** [comment]
Awesome! Please, let me know if I can help you with any grunt work. I'm happy to pitch in. :)
